kayenta-judge/src/main/scala/com/netflix/kayenta/judge/config/NetflixJudgeConfiguration.java

```diff
@@ -38,28 +30,4 @@
   NetflixJudgeConfigurationProperties netflixJudgeConfigurationProperties() {
     return new NetflixJudgeConfigurationProperties();
   }
-
```

Comment:
These beans are already setup in `kayenta-core`, moreover it is not `kayenta-judge` responsibility to create these beans, so removed them completely from here.